src/components/Account/Account.tsx

```diff
@@ -73,10 +74,14 @@ const Account = ({ panelClassName, panelHeaderClassName }: Props): JSX.Element =
 
   function translateErrors(errors?: string[]): FormErrors {
     const formErrors: FormErrors = {};
-
     // Some errors are combined in a single CSV string instead of one string per error
     errors
-      ?.flatMap((e) => e.split(','))
+      ?.flatMap((e) => {
+        if (typeof e === 'object') {
```

Comment:
`errors` is a string array so this should never happen.  Probably handling and converting the error type to string needs to happen in the services for this method.

Reply:
yes, nice catch, but this can happen. For example [here](https://github.com/jwplayer/ott-web-app/blob/2aae0ae3f98a6cad1831880258e95ed6857fbcd9/src/stores/AccountController.ts#L119), if somehow the controller throws an error, the format of the error will be Object.
In this case, the error will be `Uncaught TypeError: e.split is not a function`.

How to reproduce this:
In the [User.tsx](https://github.com/jwplayer/ott-web-app/blob/2aae0ae3f98a6cad1831880258e95ed6857fbcd9/src/pages/User/User.tsx#L101), set `canUpdateEmail` to true and try to update the email.
The controller will throw an error:
```
    if (Object.prototype.hasOwnProperty.call(values, 'email') && !canUpdateEmail) {
      throw new Error('Email update not supported');
    }
```

But, yes, I agree that this is a good starting point for post mvp refactoring

Reply:
Ah, thanks for the repro. I think I see where this needs to be fixed:
https://github.com/jwplayer/ott-web-app/blob/96f6da3bae24a7ebe5f45ff9ec958d0e470c7a90/src/components/Form/FormSection.tsx#L104

Probably something like this:
```
response = { errors: Array.of(error instanceof Error ? error.message : error as string) };
```

Reply:
OK, fixed :)